addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -292,5 +292,85 @@ registry.category("web_tour.tours").add("MergeTableTour", {
+                trigger: 'a.dropdown-item:contains("Edit Plan")',
+            },
+            {
+                content: `select linked table`,
+                trigger: 'div.isLinked div.label:contains("4")',
+            },
+            {
+                content: `unlink in edit plan if unlink possible`,
+                trigger: '.edit-buttons button:contains("Unlink")',
+            },
+            Chrome.clickMenuButton(),
+            {
+                content: `click on Edit Plan in the burger menu`,
+                trigger: 'a.dropdown-item:contains("Edit Plan")',
+            },
+            ...checkMergeTableIsCancelHelpers(),
+            Chrome.clickMenuButton(),
+            {
+                content: `click on Edit Plan in the burger menu`,
+                trigger: 'a.dropdown-item:contains("Edit Plan")',
+            },
+            FloorScreen.clickTable("4"),
+            FloorScreen.ctrlClickTable("5"),
+            {
+                content: `link in edit plan if link possible`,
+                trigger: '.edit-buttons button:contains("Link")',
+            },
+            Chrome.clickMenuButton(),
+            {
+                content: `click on Edit Plan in the burger menu`,
+                trigger: 'a.dropdown-item:contains("Edit Plan")',
+            },
+            {
+                content: `Verify table 4 and 5 is merge`,
+                trigger: 'div.table div.label:contains("4")',
+                isCheck: true,
+                run: () => {
+                    if ($("div.table div.label:contains('4')").length < 2) {
+                        throw new TourError("Table isn't merge");
+                    }
+                },
+            },
+        ].flat(),
+});
+
+registry.category("web_tour.tours").add("MergeTableEditModeTour", {
+    test: true,
+    url: "/pos/ui",
+    steps: () =>
+        [
+            Dialog.confirm("Open session"),
+            Chrome.clickMenuButton(),
+            {
+                content: `click on Edit Plan in the burger menu`,
+                trigger: 'a.dropdown-item:contains("Edit Plan")',
+            },
+            FloorScreen.clickTable("4"),
+            FloorScreen.ctrlClickTable("5"),
+            {
+                content: `link in edit plan if link possible`,
+                trigger: '.edit-buttons button:contains("Link")',
+            },
+            Chrome.clickMenuButton(),
+            {
+                content: `click on Edit Plan in the burger menu`,
+                trigger: 'a.dropdown-item:contains("Edit Plan")',
+            },
+            {
+                content: `Verify table 4 and 5 is merge`,
+                trigger: 'div.table div.label:contains("4")',
+                isCheck: true,
+                run: () => {
+                    if ($("div.table div.label:contains('4')").length < 2) {
+                        throw new TourError("Table isn't merge");
+                    }
+                },
+            },
```

Comment:
now that you put these steps in the existing tour you don't need this one, right?

Reply:
indeed :D